ci-automation/vms.sh

```diff
@@ -150,6 +152,7 @@ function _vm_build_impl() {
 
     create_digests "${SIGNER}" "images/latest/"*
     sign_artifacts "${SIGNER}" "images/latest/"*
+    ( cd images/latest ; ln -s flatcar_production_openstack_image.img.bz2 flatcar_production_brightbox_image.img.bz2 )
```

Comment:
I'm wondering if this should be done before digests and signatures are created, so we still have digests and signatures for the brightbox image.

Reply:
Thanks, pushed